packages/account-v2/package.json

```diff
@@ -11,7 +11,7 @@
         "start": "rimraf dist && npm run test && npm run serve"
     },
     "dependencies": {
-        "@deriv-com/ui": "0.0.1-beta.8",
+        "@deriv-com/ui": "0.0.1-beta.9",
```

Comment:
@thisyahlen-deriv one question, do we need to update `package-lock.json` file from **CORE** package when we change dependencies in other packages?

Reply:
i dont think so man